cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -11,3 +25,49 @@ var getCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(getCmd)
 }
+
+func preRunPackages(cmd *cobra.Command, args []string) error {
+	if !features.IsActive(features.CuratedPackagesSupport()) {
+		return fmt.Errorf("this command is currently not supported")
+	}
+	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+		if err := viper.BindPFlag(flag.Name, flag); err != nil {
+			log.Fatalf("Error initializing flags: %v", err)
+		}
+	})
+	return nil
+}
+
+func getResources(ctx context.Context, resourceType string, output string, args []string) error {
```

Comment:
Yes the reason is because packageBundles and packages have similar code except the resourcetype. This is consistent with any future get operations. All the operations will differ by only the resourcetype to get.

Reply:
But when you run `-h`, we want the supported resources to get printed right? Which we make use of cobra's functionality to do that

Reply:
Disregard above comment, misunderstood that we were just refactoring